tutorials/04-switching-physics-engines.md

```diff
@@ -0,0 +1,129 @@
+\page switchphysicsengines Switching physics engines
```

Comment:
This tutorial looks very similar to this one: https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/tutorials/physics.md

Since it's `ign-gazebo` specific, maybe we should just keep that one.